test/parallel/test-http-client-readable.js

```diff
@@ -27,11 +27,11 @@ FakeAgent.prototype.createConnection = function createConnection() {
   };
 
   // Blackhole
-  s._write = function write(data, enc, cb) {
+  s._write = function _write(data, enc, cb) {
     cb();
   };
 
-  s.destroy = s.destroySoon = function destroy() {
+  s.destroy = s.destroySoon = function destroySoon() {
```

Comment:
This one is ambiguous. I would keep `destroy` if I had the choice. Did ESLint complain?
Anyway that's fine. It's just a test...